packages/frontend/src/components/SettingsSemanticLayer/DbtSemanticLayerForm.tsx

```diff
@@ -95,13 +100,38 @@ const DbtSemanticLayerForm: FC<Props> = ({
                     label={
                         <Group display="inline-flex" spacing="xs">
                             Service Token
-                            <Tooltip
-                                maw={400}
-                                label="Service tokens can be found in your dbt Cloud account settings: https://cloud.getdbt.com/next/settings - token needs at least 'semantic layer only' permissions."
-                                multiline
+                            <HoverCard
+                                width={400}
+                                withinPortal
+                                position="top"
+                                withArrow
                             >
-                                <MantineIcon icon={IconHelp} color="gray.6" />
-                            </Tooltip>
+                                <HoverCard.Target>
+                                    <MantineIcon
+                                        icon={IconHelp}
+                                        color="gray.6"
+                                    />
+                                </HoverCard.Target>
+                                <HoverCard.Dropdown
+                                    style={{
+                                        backgroundColor: theme.colors.dark[6],
+                                    }}
```

Comment:
Don't think this is necessary. We don't have proper design guidelines, so I would just refrain from making non-standard style changes.